external-clients.rst

```diff
@@ -10,30 +10,32 @@ Introduction
    :alt: Gateway
    :align: center
 
-Netmaker allows for "external clients" to reach into a network and access services via an Ingress Gateway. So what is an "external client"? An external client is any machine which cannot or should not be meshed. This can include:
+Netmaker allows for "external clients" to reach into a network and access services via a Remote Access Gateway (ingress). So what is an "external client"? An external client is any machine which cannot or should not be meshed. This can include:
         - Phones
         - Laptops
         - Desktops
+        - Other devices that support WireGuard
 
-An external client is not "managed," meaning it does not automatically pull the latest network configuration or push changes to its configuration. Instead, it uses a generated WireGuard config file to access the designated **Ingress Gateway**, which **is** a managed server (running netclient). This server then forwards traffic to the appropriate endpoint, acting as a middle-man/relay.
+An external client is not "managed," meaning it does not automatically pull the latest network configuration or push changes to its configuration. Instead, it uses a generated WireGuard config file to access the designated **Remote Access Gateway**, which **is** a managed server (ie: a machine running netclient). This server then forwards traffic to the appropriate endpoint, acting as a middleman/relay.
```

Comment:
we should replace the word `external client`, say something like the VPN client  of the Remote Access Gateway